src/components/button.css

```diff
@@ -0,0 +1,47 @@
+:root {
+    /*Colors can be asigned per variable name here*/
+    --white: #ffffff;
+    --download: #37c8ab;
```

Comment:
use colours that align with our colour scheme! maybe one light button with dark text and one dark button with light text?